charts/cluster-autoscaler/Chart.yaml

```diff
@@ -13,8 +13,8 @@ maintainers:
     name: gjtempleton
   - email: scott.crooks@gmail.com
     name: sc250024
-name: cluster-autoscaler-chart
+name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 1.1.1
+version: 2.0.0
```

Comment:
Hmm.. unclear if this should be 9.0.0 or something, tho.  Since the last version was 8.0.0 before the chart took a detour through the other name.

Reply:
Yeah, that's a good point and would probably make migration from the deprecated chart easier on users.

I'll update this to 9.0.0 and update the docs to make clear why this is happening as well.